### PR TITLE
Add LHCb HDF5 proposal

### PR DESCRIPTION
**Pull request by apuignav**
Missing final details.

### CODE REVIEW

**Comment by apuignav** [comment]
This is ready to go

**Comment by eduardo-rodrigues** [comment]
Just "tagging me" for info - @eduardo-rodrigues.

File: _gsocproposals/2018/proposal_LHCbHDF5.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Integration of the HDF5 file format within the LHCb analysis framework
+layout: gsoc_proposal
+project: LHCb
+year: 2018
+organization: CERN
```
Comment (hegner) [inline _gsocproposals/2018/proposal_LHCbHDF5.md]: please put in all participating organizations. You can provide multiple ones as done e.g. [here](https://raw.githubusercontent.com/HEP-SF/hep-sf.github.io/master/_gsocproposals/2018/proposal_ATLASYampl.md).

File: _gsocproposals/2018/proposal_LHCbHDF5.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* [Albert Puig](mailto:albert.puig@cern.ch)
+* [Eduardo Rodrigues](mailto:Eduardo.Rodrigues@cern.ch)
+
+
+## Links
+ * [Gaudi](http://gaudi.web.cern.ch/gaudi/)
+ * [hdfgroup.org](https://www.hdfgroup.org/)
+
+
+## Corresponding project
```
Comment (etejedor) [inline _gsocproposals/2018/proposal_LHCbHDF5.md]: @hegner I believe this section is added automatically? So no need to be in the source .md file

File: _gsocproposals/2018/proposal_LHCbHDF5.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+ * [LHCb](http://hepsoftwarefoundation.org/gsoc/projects/2018/project_LHCb.html)
+
+
+## Participating organisations
```
Comment (etejedor) [inline _gsocproposals/2018/proposal_LHCbHDF5.md]: Same comment as for "Corresponding project", no need to be in the source file.

File: _gsocproposals/2018/proposal_LHCbHDF5.md (hunk 1 above)
Comment (eduardo-rodrigues) [reply _gsocproposals/2018/proposal_LHCbHDF5.md]: OK. Should be OK now.

**Comment by apuignav** [comment]
Should be fine now!

**Comment by etejedor** [comment]
Thank you Albert for the changes, I noticed something else: could you add a description for the Cincinnati and Zurich (is it ETH?) organisations here:
https://github.com/HEP-SF/hep-sf.github.io/tree/master/_gsocorgs/2018

Also an image for both of them would be needed here:
https://github.com/HEP-SF/hep-sf.github.io/tree/master/images

**Comment by apuignav** [comment]
Done! Sorry for the mess!

**Pull request merged**